tests/protocols/test_http2.py

```
@@ -0,0 +1,1031 @@
+
+    negotiator.data_received(b"GET / HTTP/1.1\r\nHost: example.org\r\n\r\n")
+    await loop.run_one()
+
+    assert isinstance(transport.get_protocol(), ZttpProtocol)
+    assert b"HTTP/1.1 200 OK" in transport.buffer
+    assert b"Hello, world" in transport.buffer
+
+
+async def test_tls_without_alpn_selects_http1():
+    app = Response("Hello, world", media_type="text/plain")
+    negotiator, transport, loop = get_negotiator(app, sslcontext=True)
```

Ruff (0.16.1)

[warning] Unpacked variable `negotiator` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)